dts/bindings/sensor/st,iis2iclx-common.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+    drdy-gpios:
+      type: phandle-array
+      required: false
+      description: DRDY pin
+
+        This pin defaults to active high when produced by the sensor.
+        The property value should ensure the flags properly describe
+        the signal that is presented to the driver.
+
+    int-pin:
+      type: int
```

Comment:
It's out of the PR's intended scope, and the same problem occurs elsewhere so won't block on this, but:

In another PR please review all uses of similar properties in ST devicetree bindings and make this and the others you'll find an `enum` too, like `st,iis2dlpc`'s `drdy-int` property.

Reply:
@pabigot 
Yes, that's a good point. I'll do it in a separate PR.
Thanks